source/deploying-with-puppet/wazuh-puppet-module/index.rst

```diff
@@ -144,5 +144,5 @@ Reference Wazuh puppet
     :maxdepth: 1
 
     reference-wazuh-puppet/ossec-scanpaths
-    reference-wazuh-puppet/wazuh-agent-class
+    reference-wazuh-puppet/wazuh-manager-class
```

Comment:
The `wazuh-manager-class` should override `wazuh-server-class` and `wazuh-agent-class` should be left intact.

Reply:
It is my fault.
Fixed.

Reply:
Great thanks